Nos/Models/CoreData/Author+CoreDataClass.swift

```diff
@@ -53,6 +53,10 @@ import Logger
         nip05?.hasSuffix("@nos.social") == true
     }
 
+    var hasMastodonNIP05: Bool {
```

Comment:
Technically we're checking for mostr.pub here -- could we rename the computed property?
```suggestion
    var hasMostrNIP05: Bool {
```

My concern is that [mostr.pub](https://mostr.pub) has more than just Mastodon (and in fact, I think we're displaying the ActivityPub logo, not the Mastodon one).